### PR TITLE
Pact test for global search

### PR DESCRIPTION
**Pull request by DavidJayakumar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-8128

### Change description ###

Add pact test for global search

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: api/test/pact/pact-tests/wa2/getSearchResults.spec.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+import * as sinon from 'sinon';
+import { mockReq, mockRes } from 'sinon-express-mock';
+import { PactTestSetup } from '../settings/provider.mock';
+import { getCcdDataAPIOverrides } from '../utils/configOverride';
+import { requireReloaded } from '../utils/moduleUtil';
+
+const { Matchers } = require('@pact-foundation/pact');
+const { somethingLike } = Matchers;
+
+const pactSetUp = new PactTestSetup({ provider: 'ccdDataStoreAPI_search', port: 8000 });
```
Comment (connorpgpmcelroy) [inline api/test/pact/pact-tests/wa2/getSearchResults.spec.ts:12]: Not sure if broker address is right, consistent with our understanding via confluence page but not consistent with https://pact-broker.platform.hmcts.net/?search=ccdData. Maybe ask Mo or Sreekanth about it? Let me know if they have an answer.

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM, apart from minor comment

File: api/test/pact/pact-tests/wa2/getSearchResults.spec.ts (hunk 1 above)
Comment (DavidJayakumar) [reply api/test/pact/pact-tests/wa2/getSearchResults.spec.ts:12]: Sreekanth  said "This is fine if test is passing locally. The broker name mapping will be in another file and will be mapped if there is some issue during deployment"

**Review by MateiHMCTS — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Pull request merged**